src/app/containers/ConsentBanner/Banner/cookie.amp.jsx

```diff
@@ -155,12 +157,12 @@ const OptionsItem = styled.li`
       text-decoration: underline;
     }
   }
-`;
 
-const ManageSettingsButton = styled.button`
-  background: none;
-  border: ${BORDER_WIDTH} solid ${C_CONSENT_ACTION};
-  color: ${C_CONSENT_ACTION};
+  button[on='tap:AMP.setState({ isManagingSettings: true })'] {
```

Comment:
Is there a reason why we need to target the buttons `on` attribute?

Reply:
Normally we would create a `StyledButton` using a `styled.button` declaration, but that also creates a new component and limits our control over style specificity. And it wouldn't work here, because this particular button (and others on the page) inherit other styles.

The `on` attribute's value is unique to this button, so we use the `[attribute]` selector to apply styles to it at the same level as the other styles on the lines above.

Reply:
`styled` creates a component and applies a unique classname to it, which references the styles you set. If that component is also inheriting other styles - the only way to override them in the cascade is to do that at the same level. That's why here you see the equivalent of `ul > button { styles }`.